jaxrs/pom.xml

```diff
@@ -64,14 +69,20 @@
         <dependency>
             <groupId>org.eclipse.microprofile</groupId>
             <artifactId>microprofile</artifactId>
-            <version>${microprofile.version}</version>
+            <version>5.0</version>
```

Comment:
Why not use the property any more?

Reply:
You are right, going to use the property, I forgot to restore it